tests/Unit/Core/EpisodeAction/EpisodeActionReaderTest.php

```diff
@@ -4,11 +4,31 @@
 namespace OCA\GPodderSync\Tests\Unit\Core\EpisodeAction;
 
 use OCA\GPodderSync\Core\EpisodeAction\EpisodeActionReader;
+use OCA\GPodderSync\Db\EpisodeAction\EpisodeActionMapper;
+use OCA\GPodderSync\Db\EpisodeAction\EpisodeActionRepository;
+use OCP\Http\Client\IClientService;
+use OCP\ICacheFactory;
 use Test\TestCase;
 
 class EpisodeActionReaderTest extends TestCase {
-	public function testCreateFromArray(): void {
-		$reader = new EpisodeActionReader();
+    /** @var IClientService */
+    private $clientService;
+
+    /** @var ICacheFactory */
+    private $iCacheFactory;
+
+    /** @var EpisodeActionRepository */
+    private $episodeActionRepository;
+
+    protected function setUp(): void {
+        parent::setUp();
+        $this->clientService = $this->createMock(IClientService::class);
```

Comment:
please add tests for new/changed code. mocking all dependencies without testing anything just makes the existing tests pass but does not tell us if things work as expected

Reply:
Same as https://github.com/thrillfall/nextcloud-gpodder/pull/114#discussion_r1117890666